.github/workflows/workspace_publish.yml

```diff
@@ -38,10 +36,5 @@ jobs:
       - name: Type check
         run: deno task test
 
-      - name: Publish (dry run)
-        if: github.event_name == 'push'
-        run: deno publish --allow-dirty --dry-run
-
       - name: Publish (real)
-        if: github.event_name == 'release'
```

Comment:
I think we want to keep this one to only publish on GH release

Reply:
This line isn't needed. See the `on` property in the file.